modules/cookbooks/pages/kafka_migrator.adoc

```diff
@@ -0,0 +1,303 @@
+
+== Setup
+
+First, we will need two clusters and, for simplicity, we can run two Redpanda Docker containers via https://docs.docker.com/compose[Docker Compose^]:
+
+[source,yaml]
+----
+# docker-compose.yml
+
+services:
+  source:
+    image: redpandadata/redpanda
+    command:
+      - redpanda
+      - start
+      - --node-id 0
+      - --mode dev-container
+      - --set rpk.additional_start_flags=[--reactor-backend=epoll]
+      - --set redpanda.auto_create_topics_enabled=false
+      - --kafka-addr 0.0.0.0:9092
+      - --advertise-kafka-addr localhost:9092
+      - --schema-registry-addr 0.0.0.0:8081
+    ports:
+      - 8081:8081
+      - 9092:9092
+      - 9644:9644
+
+  sink:
+    image: redpandadata/redpanda
+    command:
+      - redpanda
+      - start
+      - --node-id 0
+      - --mode dev-container
+      - --set rpk.additional_start_flags=[--reactor-backend=epoll]
+      - --set redpanda.auto_create_topics_enabled=false
+      - --kafka-addr 0.0.0.0:9093
+      - --advertise-kafka-addr localhost:9093
+      - --schema-registry-addr 0.0.0.0:8081
+    ports:
+      - 8082:8081
+      - 9093:9093
+      - 9645:9644
+----
+
+[source,console]
+----
+$ docker-compose -f docker-compose.yml up --force-recreate -V
+----
+
+Once these are up and running, we can use the Redpanda https://docs.redpanda.com/current/get-started/rpk[`rpk` CLI tool^] to interact with them and create two topics and a schema for each of them in the `source` cluster. Also, we create a role and, for each topic, we add an ACL.
+
+[source,console]
+----
+$ rpk -X brokers=localhost:9092 -X admin.hosts=localhost:9644 topic create foo -p 2
+$ rpk -X brokers=localhost:9092 -X admin.hosts=localhost:9644 topic create bar -p 2
+$ curl -X POST -H "Content-Type: application/vnd.schemaregistry.v1+json" --data '{"schema": "{\"name\": \"Foo\", \"type\": \"record\", \"fields\": [{\"name\": \"data\", \"type\": \"int\"}]}"}' http://localhost:8081/subjects/foo/versions
+$ curl -X POST -H "Content-Type: application/vnd.schemaregistry.v1+json" --data '{"schema": "{\"name\": \"Bar\", \"type\": \"record\", \"fields\": [{\"name\": \"data\", \"type\": \"int\"}]}"}' http://localhost:8081/subjects/bar/versions
+$ rpk -X brokers=localhost:9092 -X admin.hosts=localhost:9644 security role create foobar
+$ rpk -X brokers=localhost:9092 -X admin.hosts=localhost:9644 security acl create --operation read --topic foo --allow-role foobar
+$ rpk -X brokers=localhost:9092 -X admin.hosts=localhost:9644 security acl create --operation read --topic bar --deny-role foobar
+----
+
+Now let's use Connect to generate some AVRO-encoded messages and push them to the two topics we just created.
```

Comment:
```suggestion
Now let's use Redpanda Connect to generate some Avro-encoded messages and push them to the two topics we just created.
```

Reply:
I copied the wording from the blog post, which I hope is clear enough:

> Use Redpanda Connect to generate some Avro-encoded messages and push them to the two topics you just created.

There's a paragraph under the config which says:

> You can start this pipeline and leave it running.

Hopefully, it's clear they need to have redpanda-connect installed locally for this given that it's a cookbook in the Redpanda Connect docs.